packages/morsmall/morsmall.0.2.0/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+
+synopsis: "A concise AST for POSIX shell"
+description: """
+A concise AST for POSIX shell
+"""
+
+maintainer: "Nicolas Jeannerod <nicolas.jeannerod@irif.fr>"
+authors: [ "Nicolas Jeannerod <nicolas.jeannerod@irif.fr>" ]
+license: "GPL3"
+
+homepage: "https://github.com/colis-anr/morsmall"
+bug-reports: "https://github.com/colis-anr/morsmall/issues"
+dev-repo: "git+ssh://git@github.com/colis-anr/morsmall.git"
+
+depends: [
+  "dune"          {build}
+  "morbig"        {>= "0.10.0"}
+  "ocaml"         {>= "4.04"}
+  "ppx_deriving"  {build}
```

Comment:
This could have (now or in the future) a runtime, it should not be tagged `buil` if it is used in morsmall

Reply:
Is that always so? PPXes without runtimes shouldn't be tagged `build` in case they end up having one? Shouldn't we just change that in case it happens?

Reply:
there seems to be runtime dependencies for both `visitors` and `ppx_deriving` already:
* https://gitlab.inria.fr/fpottier/visitors/blob/master/src/META
* https://github.com/ocaml-ppx/ppx_deriving/blob/master/pkg/META.in

Reply:
OK, fair enough, I just pushed this.